arches/app/models/graph.py

```diff
@@ -118,6 +118,10 @@ def __init__(self, *args, **kwargs):
                 for card in args[0]["cards"]:
                     self.add_card(card)
 
+                for spatial_view in args[0]["spatial_views"]:
+                    spatial_view = models.SpatialView(**spatial_view)
+                    spatial_view.save()
```

Comment:
`SpatialView` is one of the only arches models with a `clean()` method. This was to move validation logic out of the views and into the models, as well as to put it in a standard place that can be called by the django admin (this is also one of the few models that's enabled in the admin).

For that reason, I think `SpatialView` should follow the Django-recommended pattern[1], which is that when saving an object, unless you're going through a form or a (de)serializer that does something equivalent for you, you call `full_clean()` to get informative, localized failures before potentially failing at the db level.

(It will surely be a long time before we update other models to follow this pattern. We have follow-ups like #11965 for some of these validation gaps. But for a model that already has an intelligent `clean()` method I think it's time to use it 👍 )
```suggestion
                    spatial_view.full_clean()
                    spatial_view.save()
```

[1]: https://docs.djangoproject.com/en/5.2/ref/models/instances/#django.db.models.Model.full_clean